doc/radosgw/notifications.rst

```diff
@@ -273,7 +273,8 @@ pushed or pulled using the pubsub sync module.
                    "eTag":"",
                    "versionId":"",
                    "sequencer": "",
-                   "metadata":""
+                   "metadata":"",
```

Comment:
I think that for metadata and tags we should add exactly how to add the Key values in the XML

Reply:
the xml structure is documented: https://docs.ceph.com/docs/master/radosgw/s3/bucketops/#create-notification
but you can use boto3 or aws CLI by adding the updated extensions file.
instructions are here: https://github.com/ceph/ceph/blob/70047a93e3d70eeb6a5db7a291cd70a9f5cec1a2/examples/boto3/README.md